### PR TITLE
Twitter pluggin uses opengraph data

### PR DESCRIPTION
**Pull request by tallowen**
Twiter values are configurable with the open graph metadata options as
described in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ Name         | Value
 data-size    | Controls the size of the share icons (bubbles/small-bubbles)
 data-tweet-at | Controls which twitter account tweeted the content (default: @firefox)
 
+Some values are not configurable through the plugin. For example the tweet text and tweet website are controled by the open graph meta data:
+
+```html
+<meta property="og:title" content="Owen is awesome" />
```
Comment (tofumatt) [inline README.md]: Pretty sure the trailing slash isn't needed nowadays.

Comment (rik) [inline README.md]: Actually, it has never been needed, except if you send Content-type: application/xhtml+xml.

Comment (tofumatt) [inline README.md]: Well, I used to set it when I was a starry-eyed kid who dreamed of the days of XHTML 2.0.

File: src/SocialShare.js [hunk 2]
```diff
@@ -147,15 +147,26 @@
...
         $('html').on('click.dropdown.data-api', clear_menus);
         $('body').on('click.dropdown.data-api', toggle, Dropdown.prototype.toggle);
         $share_container = $(share_selector);
         var tweetby = $share_container.attr('data-tweet-at');
         if (tweetby) {
-            providers['twitter']['small-bubbles']['data-via'] = tweetby;
-            providers['twitter']['bubbles']['data-via'] = tweetby;
+            set_twitter('data-via', tweetby);
+        }
+        var $meta_title = $('meta[property="og:title"]');
```
Comment (fwenzel) [inline src/SocialShare.js:163]: what does this default to if there are no opengraph tags?

File: src/SocialShare.js [hunk 3]
```diff
@@ -147,15 +147,26 @@
...
+        if ($meta_title) {
```
Comment (tallowen) [inline src/SocialShare.js:164]: If its not set here its left as `@firefox`.

**Pull request merged**